lib/game.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './frame'
+require_relative './player'
+
+class Game
+  attr_reader :frame_number, :player_1
+
+  def initialize(player_1_name:)
+    @player_1 = Player.new(name: player_1_name)
+  end
+
+  def game_start 
+    until @player_1.frame_number == 10 do
+      p 'Put number of pins knocked down in each roll'
+      p 'if scored a strike leave roll_2 empty'
+      roll_1 = gets.chomp.to_i
+      roll_2 = gets.chomp.to_i
+      play(roll_1: roll_1, roll_2: roll_2)
+    end
```

Comment:
I think the player score should be called at some point when the game is finished. At the moment the game ends and when I call the player score I am getting an error in irb for some reason?

"Put number of pins knocked down in each roll"
"if scored a strike leave roll_2 empty"
2
2
 => nil 
3.0.0 :008 > game.player_score
Traceback (most recent call last):
        5: from /Users/chrisclement/.rvm/rubies/ruby-3.0.0/bin/irb:23:in `<main>'
        4: from /Users/chrisclement/.rvm/rubies/ruby-3.0.0/bin/irb:23:in `load'
        3: from /Users/chrisclement/.rvm/rubies/ruby-3.0.0/lib/ruby/gems/3.0.0/gems/irb-1.3.0/exe/irb:11:in `<top (required)>'
        2: from (irb):8:in `<main>'
        1: from /Users/chrisclement/Desktop/Projects/Makers/Week5/weekend_challenge/Bryn/bowling-challenge-ruby/lib/game.rb:27:in `player_score'
NoMethodError (undefined method `points' for nil:NilClass)
3.0.0 :009 >